client/student/components/program/Activity/PreviewContainer.vue

```diff
@@ -0,0 +1,25 @@
+<template>
```

Comment:
What is the point of having another component that is used exclusively as a wrapper for `<content-container>`? Why not using it directly?